core/common/src/main/java/alluxio/security/authorization/DefaultAccessControlList.java

```diff
@@ -47,6 +52,37 @@ public DefaultAccessControlList(AccessControlList acl) {
     mOtherActions = new AclActions(acl.mOtherActions);
   }
 
+  /**
+   * create a child file 's accessACL based on the default ACL.
+   * @return child file's access ACL
+   */
+  public AccessControlList generateChildFileACL() {
+    AccessControlList acl = new AccessControlList();
+    acl.mOwningUser = mOwningUser;
```

Comment:
is the `DefaultAccessControlList.mOwningUser` and `mOwningGroup` guaranteed to be the same as `AccessControlList`? What happens when we change owner for a file?

Reply:
I think we need to update the fields in `AccessControlList` and `DefaultAccessControlList` when we change owner of a file.

Reply:
but, what does that? That already happens for the access one. Nothing syncs it to the default one right?